cinema/serializers.py

```diff
@@ -69,6 +105,7 @@ class MovieSessionListSerializer(MovieSessionSerializer):
         source="cinema_hall.capacity", read_only=True
     )
     tickets_available = serializers.IntegerField(read_only=True)
+    movie_image = MovieImageSerializer(source="movie.image", read_only=True)
```

Comment:
The `movie_image` field in `MovieSessionListSerializer` is using `MovieImageSerializer`, which is designed for a `Movie` instance, not directly for an `ImageField`. Consider using a simpler field like `serializers.ImageField()` if you only need the image URL.